File: allSpicee.client/src/models/Recipe.js
import { Account } from "./Account.js"

export class Recipe {
  constructor(data) {
    this.id = data.id
    this.creatorId = data.creatorId
    this.profile = data.profile
    this.title = data.title
    this.instructions = data.instructions
    this.img = data.img
    this.category = data.category
    this.liked = data.liked || false
    this.favoriteId = data.favoriteId
  }
}

File: allSpicee/Models/Recipe.cs

namespace allSpicee.Models;

public class Recipe
{
  public int Id { get; set; }
  public string Title { get; set; }
  public string Instructions { get; set; }
  public string Img { get; set; }

  public string Category { get; set; }
  public int? FavCount { get; set; }
  public int? CalcFavCount { get; set; }

  public string CreatorId { get; set; }
  public Account Creator { get; set; }


}
public class myRecipe : Recipe
{
  public int FavoriteId { get; set; }
}